roles/openshift_control_plane/files/apiserver.yaml

```diff
@@ -0,0 +1,48 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: master-api
+  namespace: kube-system
+  labels:
+    openshift.io/control-plane: "true"
+    openshift.io/component: api
+spec:
+  restartPolicy: Always
+  hostNetwork: true
+  containers:
+  - name: api
+    image: openshift/origin:v3.9.0-alpha.4
+    command: ["/bin/bash", "-c"]
+    args:
+    - |
+      #!/bin/bash
+      set -euo pipefail
+      if [[ -f /etc/origin/master/master.env ]]; then
+        set -o allexport
+        source /etc/origin/master/master.env
+      fi
+      exec openshift start master api --config=/etc/origin/master/master-config.yaml
+    securityContext:
+      privileged: true
+    volumeMounts:
+     - mountPath: /etc/origin/master/
+       name: master-config
```

Comment:
At the end of 3.10, do you see us running off of a master-config on disk or would we have the kube-apiserver use an on-disk copy and then openshift-apiserver use a configmap?  I'm betting you know which I'd prefer.....

Reply:
Would we want to split up master config first or not?  It really depends on how comfortable you think we can be with the handoff.  And once the config map exists, do we always prefer it?

Reply:
I think running assumption is we're going to move master config into a config map always the moment an apiserver split happens, so we should go with that.